nixos/modules/services/ttys/kmscon.nix

```
@@ -112,7 +112,9 @@ in
       serviceConfig.ExecStart = [
         ""
         ''
-          ${pkgs.kmscon}/bin/kmscon "--vt=%I" ${cfg.extraOptions} --seats=seat0 --no-switchvt --configdir ${configDir} --login -- ${pkgs.shadow}/bin/login -p ${autologinArg}
+          ${pkgs.kmscon}/bin/kmscon "--vt=%I" ${cfg.extraOptions} ${
+            if !cfg.hwRender then "--no-drm" else ""
```

```suggestion
            lib.optionalString (!cfg.hwRender) "--no-drm"
```
we can simplify that a little